ui2/src/pages/listing/New.tsx

```diff
@@ -8,14 +8,18 @@ import { AssetDescription } from '@daml.js/da-marketplace/lib/Marketplace/Issuan
 import {
   RequestCreateListing,
   Service,
+  ListingTypeRequest,
 } from '@daml.js/da-marketplace/lib/Marketplace/Listing/Service';
-import { publicParty } from '../../config';
-import { ServicePageProps } from '../common';
+import { Service as MarketClearingService } from '@daml.js/da-marketplace/lib/Marketplace/Clearing/Market/Service';
+import { publicParty, usePartyName } from '../../config';
+import { ServicePageProps, createDropdownProp } from '../common';
 import { Button, Form, Header, Icon } from 'semantic-ui-react';
 import FormErrorHandled from '../../components/Form/FormErrorHandled';
 import { IconClose } from '../../icons/icons';
 import Tile from '../../components/Tile/Tile';
 
+const COLLATERALIZED_VALUE = 'COLLATERALIED_MARKET';
```

Comment:
`COLLATERALIZED`

Reply:
spelling is hard, but I guess this is why consts instead of just directly comparing strings is good...